src/kzg10/data_structures.rs

```diff
@@ -221,17 +227,20 @@ pub struct Proof<E: PairingEngine> {
 impl<E: PairingEngine> PCProof for Proof<E> {
     fn size_in_bytes(&self) -> usize {
         let hiding_size = if self.random_v.is_some() {
-            algebra_core::to_bytes![E::Fr::zero()].unwrap().len()
+            to_bytes![E::Fr::zero()].unwrap().len()
```

Comment:
```suggestion
            E::Fr::UNCOMPRESSED_SIZE
```

Reply:
This is a bit weird. The hiding size uses the uncompressed Fr size, while the G1Affine uses the compressed one (since it divides by 2?)?